### PR TITLE
Preparing for dependency on nupic.bindings 0.2.3

### PR DESCRIPTION
**Pull request by rhyolight**
Do not merge. This will get merged just before a NuPIC 0.3.7 release. This PR is for switching to use the latest nupic.bindings version and prepping for next release.

### CODE REVIEW

**Comment by rhyolight** [comment]
This is going to fail CI until nupic.bindings 0.2.3 is available.

File: external/common/requirements.txt [hunk 1]
```diff
@@ -13,9 +13,9 @@ unittest2==0.5.1
 validictory==0.9.1
 PyMySQL==0.6.2
 DBUtils==1.1
-pyproj==1.9.3
+pyproj>=1.9.3
```
Comment (rhyolight) [inline external/common/requirements.txt]: @rcrowder Why is the `<=` required here?

Comment (rcrowder) [inline external/common/requirements.txt]: PyProj 1.9.3 has been replaced on Windows with 1.9.4 Building 1.9.3 from source is problematic. Similar to why a numpy 1.9.1 prebuilt package is used.

**Comment by rhyolight** [comment]
Closing in favor of #2875

**Pull request closed without merging**